charts/rancher-gke-operator-crd/rancher-gke-operator-crd/2.0.100-rc1+up2.0.1-rc1/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/namespace: cattle-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/release-name: rancher-gke-operator-crd
+apiVersion: v2
+appVersion: 2.0.1-rc1
```

Comment:
Is this right? Before this we would have had a generated-changes patch to make this `2.0.1`.